.detoxrc.json

```diff
@@ -28,7 +28,7 @@
     "simulator": {
       "type": "ios.simulator",
       "device": {
-        "type": "iPhone 11"
+        "type": "iPhone 14"
```

Comment:
I found that this was needed in order for tests to work on EAS, otherwise there is a `no such simulator` error from `applesimutils`.